.github/actions/deploy-template-operator-with-modulereleasemeta/action.yml

```diff
@@ -79,10 +79,9 @@ runs:
         sed -i 's/k3d-private-oci-reg.localhost:5001/private-oci-reg.localhost:5000/g' ./template.yaml
         kubectl get crds
         kubectl apply -f template.yaml
-    - name: Create and apply ModuleReleaseMeta from the latest release
+    - name: Create and apply ModuleReleaseMeta from the template-operator repo
       working-directory: template-operator
       if: ${{ matrix.e2e-test == 'kyma-metrics' ||
-        matrix.e2e-test == 'non-blocking-deletion' ||
```

Comment:
Quick info on why this testcase is not important anymore? I don't know the details.

Reply:
check this comment: https://github.com/kyma-project/lifecycle-manager/pull/2736#discussion_r2447030150